sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -735,4 +738,39 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  test("broadcast hint on temp view") {
```

Comment:
Seems this test is only for permanent views. Maybe we can put everything in one test in https://github.com/apache/spark/pull/27935/files#diff-fa1d044f9cfe587e27866393fe18fd46R329

Reply:
ok

Reply:
Moved this test into `DataFrameJoinSuite`.